backend/core/detection.py

```diff
@@ -589,6 +591,9 @@ def detect_path_type_N_hijack(
             self, monitor_event: Dict, prefix_node: radix.Radix, *args, **kwargs
         ) -> Tuple[int, str]:
             # Placeholder for type-N detection (not supported)
+            for item in prefix_node.data["confs"]:
+                if item["no-export"]:
+                    return (monitor_event["path"][-3], "N")
```

Comment:
Not sure if hijacker is 1st neighbor or 2nd neighbor

Reply:
Also, we need to have a list of monitoring ASes that actually can announce no-export prefixes. That case would be:
`origin AS -> neighbor AS -> monitoring AS`

Reply:
this is hard since it is dynamic and we cannot be sure. For now could you simply add a check to return this only if the path has length > 2 (origin + neighbor)? We assume that paths with length 2 are ok.

Reply:
> Not sure if hijacker is 1st neighbor or 2nd neighbor

We assume the first neighbor for now since this is the one that leaked the route upstream. So [-2]

Reply:
So [-3] is correct